core/services/bulletprooftxmanager/nonce_syncer.go

```diff
@@ -109,7 +101,7 @@ func (s NonceSyncer) SyncAll(ctx context.Context) (merr error) {
 	return errors.Wrap(merr, "NonceSyncer#fastForwardNoncesIfNecessary failed")
 }
 
-func (s NonceSyncer) fastForwardNonceIfNecessary(ctx context.Context, address common.Address) error {
+func (s NonceSyncer) fastForwardNonceIfNecessary(ctx context.Context, address common.Address) (err error) {
```

Comment:
probably no reason to name `(err error)` as it's being shadowed in the next line

Reply:
Good catch, this was left over from a previous experiment

Reply:
weird that static checks sometimes seem to detect that, but not always